pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -214,6 +218,7 @@ private static long getRandomInitialDelayInSeconds() {
 
     private static final int DEFAULT_SEGMENT_LEVEL_VALIDATION_INTERVAL_IN_SECONDS = 24 * 60 * 60;
     private static final int DEFAULT_SEGMENT_RELOCATOR_FREQUENCY_IN_SECONDS = 60 * 60;
+    private static final int DEFAULT_SEGMENT_TIER_ASSIGNER_FREQUENCY_IN_SECONDS = 60 * 60;
```

Comment:
We might want to put a longer interval because it will read all segments' ZK metadata

Reply:
making it 24hr